guides/doc-Administering_Red_Hat_Satellite/master.adoc

```diff
@@ -17,6 +17,10 @@ include::common/assembly_using-satellite-ansible-content-collections.adoc[levelo
 
 include::topics/Users_and_Roles.adoc[]
 
+ifdef::katello,satellite[]
```

Comment:
```suggestion
ifdef::katello,satellite,orcharhino[]
```

Reply:
Can anyone explain to me why we're excluding foreman-el here? 
I am unsure if we even need an ifdef. It seems to apply to all scenarios?

Reply:
I can't.
If it's in "using Puppet for configuration management", then you'd need to apply the include logic there anyways. Does that work for Satellite?

Reply:
Puppet support is remaining for Satellite to some degree, I don't fully understand it. 
@ehelms can you help us understand a bit better. 
First question: we can just drop the ifdef as this applies everywhere I can tell?
Second question: We have quite an old outdated Satellite guide here: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/puppet_guide/index - while recently @maximiliankolb created these very nice config management docs here: https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#using-puppet-for-configuration-management_managing-hosts
My question: if there is some degree of Puppet supported for Satellite, could we not just expose these sections for Satellite also?

Reply:
Currently, for foreman-el, Puppet is still enabled by default. So enabling Puppet in this way would not apply to those users. This only applies to Katello/Orcarhino/Satellite for the present moment.

Reply:
Thank you !

Reply:
Houston, we have a problem.
The "Using Puppet..." assembly is included for all products except Satellite (`ifndef::satellite[]`), so if that condition is correct, then it's a no-go for the intention I had.
I still think it would fit best in the Admin Guide, because it's about project system configuration rather than managing hosts. Enabling Puppet can be a prerequisite for the latter, and xref'd for the sake of Orcharhino and Katello.

Reply:
I see! Thanks, I'm on it.